doc/Type/Date.pod6

```diff
@@ -28,7 +28,7 @@ an object the current day according to the system clock.
     say $n - $d;                    # OUTPUT: «7␤», 7 days between New Years/Christmas Eve
     say $n + 1;                     # OUTPUT: «2016-01-01␤»
 
-B<Note> since version 6.d, .perl can be called on C<Date>. It will also
+B<Note> since version 6.d, .raku can be called on C<Date>. It will also
```

Comment:
Not sure about this one. To be precise, raku was introduced a year later. Maybe ".perl (and .raku since version 2019.22"?

Reply:
I would turn it around: the present as main statement and the past between parenthesis:
".raku (.perl before version 2019.22)".